src/containers/MyNdla/Learningpath/components/utils.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * Copyright (c) 2024-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+import config from "../../../../config";
+
+export const sharedLearningpathLink = (id: number) => `${config.ndlaFrontendDomain}/learningpath/${id}`;
```

Comment:
Kan vi ikke bruke `routes` til dette?

Reply:
Og så trur eg ikkje vi skal bruke samme path som frittstående læringgsti. Eller kanskje vi skal det for vi har vel egentlig aldri tatt det i bruk nokon plass.

Reply:
Det er samme for meg hva vi bruker 😅  Har bare valgt det som ga mest mening mens jeg koda :D